### PR TITLE
Low-hanging doc consolidation for directives

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

**Comment by jrudolph** [comment]
Wow, already great progress :+1:

File: docs/src/main/paradox/java/http/routing-dsl/directives/basic-directives/extract.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # extract
 
+@@@ div { .group-scala }
```
Comment (jonas) [inline docs/src/main/paradox/java/http/routing-dsl/directives/basic-directives/extract.md]: Why put Scala signatures in the Java version?

**Comment by jonas** [comment]
Nice

File: docs/src/main/paradox/java/http/routing-dsl/directives/basic-directives/extract.md (hunk 1 above)
Comment (raboof) [reply docs/src/main/paradox/java/http/routing-dsl/directives/basic-directives/extract.md]: Basically my strategy is to make the difference between the Java and Scala versions as small as possible until they are the same and the Java version can be replaced by a symlink

**Comment by raboof** [comment]
(rebased & fixed conflicts - should be nice to merge if green :) )

**Pull request merged**